### DISCUSSION TITLE
Question of Atlas

### DISCUSSION THREAD

**Discussion by qiudashuai1** [CoBrALab/RABIES#394] [General]
Hi, RABIES using DSURQE atlas as default atlas, but I can't open the link of https://wiki.mouseimaging.ca/display/MICePub/Mouse+Brain+Atlases. I want to know the brain atlas labels of DSURQE, can you provide me a csv file or alternative link? Another question is, if I want to use Allen CCFv3 atlas, do I need another transform?

**Comment by gdevenyi** [comment]
Hi. We are aware of the access issue with the Dorr atlas files and working with the original authors to migrate them to a more reliable hosting solution.

The CCFv3 atlas is not suitable for use as an MRI template in its current form. We will be working to provide some solutions in that space in the future.

**Reply by qiudashuai1** [reply]
Thank you for your help. However, I urgently need the label names for DSURQE to interpret the connectivity results. Do you have the label names available in a CSV or TXT file, or could you provide a link where I can download them?

**Reply by Gab-D-G** [reply]
Here a copy of the CSV
[DSURQE_40micron_R_mapping.csv](https://github.com/user-attachments/files/17675945/DSURQE_40micron_R_mapping.csv)
